frontend/packages/shared/src/hooks/useLocalStorage.ts

```diff
@@ -1,25 +0,0 @@
-import { useState, useEffect } from 'react';
-
-/*
- * @Deprecated
- * This hook is deprecated and is replaced by "import { useLocalStorage } app-shared/hooks/useWebStorage"
- */
-export const useLocalStorage = (key: string) => {
-  const [valueInStorage, setValueInStorage] = useState(localStorage.getItem(key));
-
-  useEffect(() => {
-    const handleStorageChange = (event) => {
-      if (event.key === key) {
-        setValueInStorage(event.newValue);
-      }
-    };
-    window.addEventListener('storage', handleStorageChange);
-    return () => {
-      window.removeEventListener('storage', handleStorageChange);
-    };
```

Comment:
@standeren could you please tell me why we needed to listen on the storage? Im not sure if I'm breaking any features by removing the event-listener. It was used within the AppPreview, LandingPage for `selectedLayoutInEditor`. 😊

Reply:
we needed to listen to storage to ensure that "Back to editor" button was sending the user back to the correct layoutset. I have fixed this without the need for listen to localstorage, https://github.com/Altinn/altinn-studio/pull/10829/commits/9e65b2dd08595002d977ff07c8eee0ed7129e1b7.